### DISCUSSION TITLE
Lethean AMA brief 04.12.21

### DISCUSSION THREAD

**Discussion by JPH-jph** [valiant1x/intensecoin#178] [Q&A]
**_Question:_** How many premine do you have? put an end to premine
**_Answer:_** The premine is available to us, it is kept by a reliable person, there are ~ 14kk coins left (the site will contain information about the current state of the premine), when the shift took over it was ~ 19kk (5kk compensation for current expenses, there were at least 10k GBP expenses)

**_Question:_** Plans to change the algorithm. When is the fork? Is it going to be RandomX for sure? Will we continue to follow Monero?
**_Answer:_** Exactly RandomX, when? There is no exact date, the second developer is working on it. We will follow Monero.

**_Question:_** Will there be a mobile client?
**_Answer:_** We moved away from Electron, switched to Angular & Wails, which means there will be a web (cross-platform) application - there will be both Windows and Mac and Android immediately after development is completed, with Linux you will have to wait for January-February 2022

**_Question:_** Contract with third-party projects, such as radio miners, for example helium
**_Answer:_** No comments yet 

**_Question:_** The role of the coin in the Lethean ecosystem
**_Answer:_** This is the backbone of our blockchain and the primary means of payment for the services offered by our platform. 

**_Question:_** What is most lacking for development? How can you help?
**_Answer:_** Not enough time and contributors. Anyone can help with development, writing and updating of documentation, marketing (but so far there is nothing to promote) 

**_Question:_** What are your plans to change the current size of emissions which was found to be an error? How many coins are in the block? The number of blocks per day?
**_Answer:_** We have a proposed solution and will be shared when the team is ready to pursue this. This has been independently verified to confirm our observations were correct including the approach options.

**_Question:_** What kind of certificate were we waiting for? Why didn't they wait? And what should we do with this misfortune now?
**_Answer:_** We waited for an Extended Signature (EV) certificate. It is needed to sign our applications. Without it the application will most likely not run, including the possibility of being stopped by AV and impact general usability. The certificate is received, and as soon as the applications are ready, we will be able to sign them. 

**_Question:_** Plans to change / expand the team
**_Answer:_** There are no such plans yet, but we will gladly accept development assistance, you can learn more from Snider (be sure to check our https://github.com/letheanVPN for the programming languages ​​used)

**Comment by Snider** [comment]
Thank you for writing this up @JPH-jph
